README.md

```diff
@@ -1101,6 +1122,8 @@ terragrunt = {
     before_imports   = false                          # optional, run command before terraform imports its files
     after_init_state = false                          # optional, run command after the state has been initialized
     order            = 0                              # optional, default run hooks in declaration order (hooks defined in uppermost parent first, negative number are supported)
+    env_vars         = {}                             # optional, define environment variables only available during hook execution
+    global_vars      = {}                             # optional, define global environment variables (exist while and after executing the hook)
```

Comment:
I think this is really weird. I think this should be a top level attribute (out of the hook).
Something like `persistent_env_vars` (Initiated with a value, or not). It is really badly discoverable when it is defined in the hook like that